pkg/asset/installconfig/installconfig.go

```diff
@@ -19,8 +19,8 @@ const (
 )
 
 var (
-	defaultServiceCIDR      = parseCIDR("10.3.0.0/16")
-	defaultClusterCIDR      = "10.2.0.0/16"
+	defaultServiceCIDR      = parseCIDR("172.30.0.0/16")
+	defaultClusterCIDR      = "10.128.0.0/14"
```

Comment:
Where did this default come from?  There may have been issues getting it into Terraform (I'm not sure), but I'm [seeing][1]:

```
* aws_vpc.new_vpc: Error creating VPC: InvalidVpc.Range: The CIDR '10.128.0.0/14' is invalid.
```

Maybe an AWS limitation, because [the 3.11 docs suggest this value][2]?

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/724/pull-ci-openshift-installer-master-e2e-aws/1747/build-log.txt
[2]: https://docs.openshift.com/container-platform/3.11/install_config/configuring_sdn.html#configuring-the-pod-network-on-masters

Reply:
THe clusterCIDR  is network block used to give PodIPs it has nothing to do with AWS.

Reply:
So terraform doen't need these. these are kubernetes networking options.

Reply:
> The clusterCIDR is network block used to give PodIPs it has nothing to do with AWS.

Ah, so do we have a separate CIDR config for the VPC?

Reply:
Ah, VPC CIDR is [here][1] with a `/16`, so sorry for my noise :/.

[1]: https://github.com/openshift/installer/blob/v0.4.0/pkg/asset/installconfig/aws/aws.go#L19